.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # These people will be added as reviewers to all commits.
 # Feel free to add yourself if you want.
-* @ITNano @farpoke @AntonNeld
\ No newline at end of file
+* @ITNano @farpoke @AntonNeld
```

Comment:
I will focus on the UI part, so I'll skip being on the list in this repo.

Reply:
Ok! Fixing!